### PR TITLE
Updated config and code for fractional amount

### PR DESCRIPTION
**Pull request by peterbarrow**
Commitment is now specified as a fraction.
The total commitment amount is that fraction times the total collateral for the user (bond+general+margin)

### CODE REVIEW

File: bot/normal/normal.go [hunk 1]
```diff
@@ -245,11 +246,14 @@ func (b *Bot) submitOrder(sub *api.PrepareSubmitOrderRequest) error {
 }
 
 func (b *Bot) sendLiquidityProvision(buys, sells []*proto.LiquidityOrder) error {
+	// CommitmentAmount is the fractional commitment value * total collateral
+	commitment := b.strategy.CommitmentFraction * (float64(b.balanceGeneral + b.balanceMargin + b.balanceBond))
```
Comment (ashleyvega) [inline bot/normal/normal.go:250]: comment (non-blocking): no need for the outer brackets around `(float(...))`.

File: bot/normal/normal.go [hunk 2]
```diff
@@ -304,6 +308,28 @@ func (b *Bot) getAccountMargin() error {
 	return nil
 }
 
+// getAccountMargin get this bot's bond account balance.
```
Comment (ashleyvega) [inline bot/normal/normal.go:311]: comment (non-blocking): `getAccountMargin` -> `getAccountBond`

**Review by ashleyvega — APPROVED** [review]
LGTM (2 non-blocking comments)

File: bot/normal/normal.go (hunk 1 above)
Comment (peterbarrow) [reply bot/normal/normal.go:250]: Fixed, thanks.

File: bot/normal/normal.go (hunk 2 above)
Comment (peterbarrow) [reply bot/normal/normal.go]: Fixed, thanks.

**Pull request merged**